### PR TITLE
Setup your web server(NGINX,FIREWALL,GIT,NODE,YARN)

### PR DESCRIPTION
**Pull request by abhayst007**

### CODE REVIEW

File: Basic Web Server Setup(Ubuntu)/setup.sh [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+sudo n stable
+npm -v
+
+echo "Installing Yarn!"
+npm install --global yarn
+yarn --version
+
+echo "Setup Done!"
+
+echo"Restart to start with the latest stable release of Node!"
```
Comment (suraj-singh12) [inline Basic Web Server Setup(Ubuntu)/setup.sh:34]: add a space after `echo`, else it gives error

File: Basic Web Server Setup(Ubuntu)/setup.sh [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+echo "Adjusting your firewall rules! \n So that no one can get into your system easily"
+sudo apt-get install ufw
+sudo ufw enable -y
+sudo ufw app list
+sudo ufw allow 'Nginx HTTP'
+sudo ufw status
+
+echo "Checking your web server"
+sudo systemctl status nginx
```
Comment (suraj-singh12) [inline Basic Web Server Setup(Ubuntu)/setup.sh]: holds terminal from executing the script any further, and awaits user to press `q`

**Review by suraj-singh12 — CHANGES_REQUESTED** [review]

**Comment by suraj-singh12** [comment]
Add a demo file, to show the user how to start the server.

**Review by suraj-singh12 — APPROVED** [review]

**Comment by abhayst007** [comment]
Changes made in README.md to provide some more info on how to execute the file

**Pull request merged**